CommonTools/Utils/test/testFormulaEvaluator.cc

```diff
@@ -781,7 +781,7 @@ testFormulaEvaluator::checkFormulaEvaluator() {
     std::vector<double> xValues = {.1, 1., 10., 100.};
 
 
-    auto func =[&v](double x) {return 100./3.*0.154227+2.36997;
+    auto func =[&](double x) {return 100./3.*0.154227+2.36997;
```

Comment:
Hi, doesn't having simply an ampersand mean you capture all local variables? I guess clang also doesn't like that. To capture nothing you have to do `[]` I think.

Reply:
clang doesn't actually care about `[&]` (i.e. capture needed variables by reference) it just cares if you explicitly capture a variable name but then never use it.

Reply:
Ah ok, interesting! Thanks.

Reply:
I changed to using `[]` anyway, since it is cleaner.